AirCasting/SearchAndFollow/SearchMap/SearchMapView.swift

```diff
@@ -51,22 +51,41 @@ struct SearchMapView: View {
                         .disabled(!viewModel.searchAgainButton)
                         .opacity(viewModel.searchAgainButton ? 1.0 : 0.0)
                 })
-                    .padding(.top, 50)
+                    .padding(.top, 20)
                     .padding(.horizontal)
             })
         }
             .onChange(of: viewModel.shouldDismissView, perform: { result in
                 result ? self.presentationMode.wrappedValue.dismiss() : nil
             })
             .alert(item: $viewModel.alert, content: { $0.makeAlert() })
+            .sheet(isPresented: $viewModel.isLocationPopupPresented, onDismiss: {
+                viewModel.redoTapped()
```

Comment:
shortcut detected. Should be a separate, dedicated function since it's not a "reto tapped" situation here :)